### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by alex1779**
<a name="readme-top"></a>

<div align="center">

  
  <br/>

  <h3><b>"Portfolio" project</b></h3>
  <br/>
</div>
For the second milestone in building my portfolio website, I will create the mobile website section where you will list my portfolio projects. 
Created the middle section of the mobile website.
I have added works section and about me page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,6 +54,208 @@
...
+              <li id="social-media-icons-item-3">
+                <a href="#"><img class="social" src="images/hand.svg" alt="hand" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-4">
+                <a href="https://twitter.com/AlejandroMaggi7"><img class="social" src="images/twitter.svg" alt="twitter" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-5">
+                <a href="#"><img class="social" src="images/m.svg" alt="m" width="auto" height="auto"></a>
+              </li>
+            </ul>
```
Comment (ridwanediallo) [inline index.html:209]: - [ ] Great job! 👏 However, your `LET'S CONNECT` heading and your `social-media-icons` are overlapping. Please would you mind fixing that?
Screenshot from 2022-11-29 22-45-42

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @alex1779 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (alex1779) [reply index.html:209]: @ridwanediallo Thanks! 
What browser are you using to see the error?? I'm using Google Chrome and I can't see that error. Everything seems ok by my side.

**Review by amejid — APPROVED** [review]
Hi @alex1779, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**